psi4/driver/driver.py

```diff
@@ -1693,8 +1739,18 @@ def frequency(name, **kwargs):
     if freq_mode == 'sow':
         return 0.0
 
-    wfn.frequencies().print_out()
-    core.thermo(wfn, wfn.frequencies())
+    # Project final frequencies?
+    translations_projection_sound = not core.get_option('SCF', 'EXTERN') and \
+                                    not core.get_option('SCF', 'PERTURB_H')
+    rotations_projection_sound = translations_projection_sound and wfn.gradient().rms() < 1.e-2  # aforementioned hat
```

Comment:
Similarly, the `1.0e-2` would be better if it were a variable, so that both locations are updated consistently in the event of us changing tolerance.